dspace-api/src/main/java/org/dspace/core/HibernateDBConnection.java

```diff
@@ -184,17 +184,19 @@ private void configureDatabaseMode() throws SQLException {
                 }
             }
 
+            // ITEM
             if (entity instanceof Item) {
                 Item item = (Item) entity;
 
-                if(Hibernate.isInitialized(item.getSubmitter())) {
-                    uncacheEntity(item.getSubmitter());
-                }
+                //DO NOT uncache the submitter. This could be the current eperson. Uncaching could lead to
```

Comment:
Maybe it is reasonable to merge it as it is for DSpace 6.2 and to change it for our master branch that will be tested for DSpace 7.0. And in case we don't achieve this know (many developers have vacations currently), we can merge as it is and create a ticket like "EPersons cached as submitters aren't uncached when their item is".